po/pt.po

```diff
@@ -1570,14 +1570,14 @@ msgid ""
 "A <b>summary bar</b> that displays all of the relevant account's balance "
 "information"
 msgstr ""
-"uma <b>barra de sumário</b> que mostra toda a informação relevante do saldo "
-"da conta;"
+"Uma <b>barra de sumário</b> que mostra toda a informação relevante do saldo "
+"da conta"
 
 #: features.phtml:59
 msgid "Many options to <b>customize</b> the appearance and the display style"
 msgstr ""
-"muitas opções para <b>personalizar</b> a apar&#234;ncia e o estilo de "
-"exibição."
+"Muitas opções para <b>personalizar</b> a apar&#234;ncia e o estilo de "
+"exibição"
```

Comment:
@jralls What are the common separators and end symbol in en_US for a bullet list like  before? https://www.gnucash.org/beta/features.phtml: Checkbook-Style Register; Reports, Graphs

Reply:
@fellen The bullet is usually a sufficient delimiter. If there's a heading it may end with a colon (':'). A list item will usually have a period (a.k.a. full stop) only if it is a full sentence. When writing a list without bullets one would typically use commas to separate items if they're only a word or two and semicolons (';') if they're phrases, especially ones that might include commas.